timber-lint/src/test/java/timber/lint/WrongTimberUsageDetectorTest.kt

```diff
@@ -922,7 +922,7 @@ class WrongTimberUsageDetectorTest {
                 |  }
                 |}""".trimMargin())
         )
-        .issues(*WrongTimberUsageDetector.getIssues())
+        .issues(*WrongTimberUsageDetector.issues)
```

Comment:
Hope there's a list-accepting overload

Reply:
replaced with the individual issue, so we can nix this.